webview/src/experiments/components/table/styles.module.scss

```diff
@@ -576,7 +580,7 @@ $badge-size: 0.85rem;
   height: 100%;
 
   &:first-child {
-    margin-right: 20px;
+    margin-right: 4px;
```

Comment:
[Q] Will changing this affect the way the shadow is displayed? Is there a gotcha here? Maybe with the default size of the first column?

Reply:
Change margin shouldn't affect the shadow or the width of the first column. It just affects the width of the exp name text. By decreasing the margin, the exp-name/commit has a bit more room :)